common/hooks/post-install/11-pkglint-patch-orig.sh

```diff
@@ -0,0 +1,22 @@
+# vim: set ts=4 sw=4 et:
+#
+# This hook executes the following tasks:
+#   - Looks on all packages for *.orig files generated by patch on fuzzy match
+
+hook() {
+    local matches mime file f prune_expr dir
+
+    while read -r file; do
+        basename="${file%.orig}"
+        if [ -f "$basename" ]; then
+            if [ -z "$matches" ]; then
+                matches=yes
+                msg_red "Fuzzy patches generate those files:\n"
+            fi
+            msg_red "    $file\n"
+        fi
+    done < <(find $PKGDESTDIR -type f -name '*.orig')
+    if [ "$matches" ]; then
+        msg_error "Please refresh the patches\n"
```

Comment:
This would probably break a lot of builds, if this is does get merged, probably would make more sense to have it as a strict option?: https://github.com/void-linux/void-packages/blob/2f69057b87c8e999d2d2baef19d4fea41ac62efd/common/environment/setup/vsed.sh#L61-L67